tests/integration/test_open_lineage.py

```diff
@@ -0,0 +1,87 @@
+import time
+
+import pytest
+
+from pyatlan.cache.role_cache import RoleCache
+from pyatlan.client.atlan import AtlanClient
+from pyatlan.model.assets import SparkJob
+from pyatlan.model.enums import OpenLineageEventType
+from pyatlan.model.open_lineage.event import OpenLineageEvent
+from pyatlan.model.open_lineage.job import OpenLineageJob
+from pyatlan.model.open_lineage.run import OpenLineageRun
+
+client = AtlanClient()
+
+
+@pytest.fixture(scope="module")
+def setup_connection():
+    admin_role_guid = RoleCache.get_id_for_name("$admin")
+    connection = client.open_lineage.create_connection(
+        name="my-test", admin_roles=[admin_role_guid]
+    )
+
+    yield connection
+
+    client.asset.purge_by_guid(guid=connection.guid)
+
+
+def test_open_lineage_integration(setup_connection):
```

Comment:
Please use type hints wherever possible for better readability and type safety:
```suggestion
def test_open_lineage_integration(connection: Connection):
```